radix-engine/src/model/nodes/transaction_processor.rs

```diff
@@ -228,596 +220,473 @@ impl TransactionProcessor {
         }
     }
 
-    pub fn static_main<'s, Y, R>(
-        func: TransactionProcessorFunction,
-        args: ScryptoValue,
+    pub fn static_main<'a, Y>(
```

Comment:
MInor:  Can we rename `static_main` to `run`, so to help me forget main dispatching?

Reply:
yup!